src/navigator/IFrameNavigator.ts

```diff
@@ -3228,6 +3215,18 @@ export default class IFrameNavigator implements Navigator {
         this.loadingMessage.style.display = "none";
         this.loadingMessage.classList.remove("is-loading");
       }
+      if (this.view.layout !== "fixed") {
+        console.log("hideLoadingMessage");
+        // this.checkResourcePosition();
```

Comment:
Remove commented out code and console log?

Reply:
removed